src/contrib/mlir/core/pass/ng_dialect_builder.cpp

```diff
@@ -216,6 +219,11 @@ mlir::ArrayAttr NgDialectConversionPass::getShapeAsAttr(T ngShape)
     return m_builder.getI64ArrayAttr(mlirShape);
 }
 
+mlir::Attribute NgDialectConversionPass::getIntAsAttr(int64_t value)
+{
+    return m_builder.getI64IntegerAttr(value);
+}
+
```

Comment:
Could we use `m_builder` directly instead of this utility? I think this utility is not helping too much and could give the impression that we should add wrappers for all the builder methods

Reply:
m_builder is private in the pass class NgDialectConversionPass

Reply:
Not sure I understand. This utility is also private (?)